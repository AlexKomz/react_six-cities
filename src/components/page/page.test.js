import React from "react";
import renderer from "react-test-renderer";

import Page from "./page.jsx";

const children = <div></div>;

it(`Should Page render correctly`, () => {
  const tree = renderer
    .create(
        <Page pageClasses={`page`}>
          {children}
        </Page>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
