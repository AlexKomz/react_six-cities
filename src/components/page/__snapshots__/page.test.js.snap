// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Page render correctly 1`] = `
<div
  className="page"
>
  <div />
</div>
`;
